viewing_party/party.py

```diff
@@ -1,23 +1,160 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if not title or  not genre or not rating:
+        return None
+    else:
+        movies = {}
+        movies["title"] = title
+        movies["genre"] = genre
+        movies["rating"] = rating
+        return movies 
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data,movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    if title in user_data["watchlist"]:
+        user_data["watchlist"].remove(title)
+        user_data["watched"].append(title)
+        return user_data
+    return user_data
 
 # -----------------------------------------
 # ------------- WAVE 2 --------------------
 # -----------------------------------------
 
+def get_watched_avg_rating(user_data):
+    sum_rating = 0
+    no_movies = 0

```

Comment:
The name `no_movies` could be a little confusing since the `no` could be misinterpreted. Something like `count` or `num_movies` would be a little clearer.